js/mviewer.js

```diff
@@ -463,6 +489,38 @@ mviewer = (function () {
         _map.addLayer(_overlayFeatureLayer);
     };
 
+
+    /**
+     * Private Method: initSelectOverlay
+     * this layer is used to render ol.Feature in info tool
+     */
+
+    var _initSelectOverlay = function () {
+        _sourceSelectOverlay = new ol.source.Vector();
+        _selectOverlayFeatureLayer = new ol.layer.Vector({
+            source: _sourceSelectOverlay,
+            style: getSelectStyle.bind(this, '82, 98, 217', 4)
+        });
+        _selectOverlayFeatureLayer.set('mviewerid', 'selectoverlay');
+        _map.addLayer(_selectOverlayFeatureLayer);
+    };
+
+
+    /**
+     * Private Method: initSubSelectOverlay
+     * this layer is used to render ol.Feature of sub selection in info tool
+     */
+
+    var _initSubSelectOverlay = function () {
+        _sourceSubSelectOverlay = new ol.source.Vector();
+        _subSelectOverlayFeatureLayer = new ol.layer.Vector({
+            source: _sourceSubSelectOverlay,
+            style: getSelectStyle.bind(this, '252, 186, 3', 2)
```

Comment:
It could be better to avoid a second layer if you just display a different style.

Reply:
(see above)